File: ClientApp/ClientApp/Models/ResponseErrors/LoginResponseErrors.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClientApp.Models.ResponseErrors
{
    class LoginResponseErrors
    {
        public string UsernameError { get; set; }
        public string PasswordError { get; set; }
        public string GeneralError { get; set; }
    }
}

File: M_Trivia/M_Trivia/LoginResponseErrors.cpp
#include "LoginResponseErrors.hpp"

bool LoginResponseErrors::noErrors() const {
  return passwordError.empty() && usernameError.empty() && generalError.empty();
}

nlohmann::json LoginResponseErrors::getJson() const {
  nlohmann::json j = nlohmann::json{
      {"PasswordError", passwordError},
      {"UsernameError", usernameError},
      {"GeneralError", generalError},
  };
  return j;
}
